src/Behat/Behat/Definition/Translator/TranslatorInterface.php

```diff
@@ -0,0 +1,14 @@
+<?php
+
+namespace Behat\Behat\Definition\Translator;
+
+if (interface_exists(\Symfony\Contracts\Translation\TranslatorInterface::class)) {
+    interface TranslatorInterface extends \Symfony\Contracts\Translation\TranslatorInterface
+    {
+    }
+} else {
+    /** @scrutinizer ignore-unused */
+    interface TranslatorInterface extends \Symfony\Component\Translation\TranslatorInterface
```

Comment:
As documented: https://scrutinizer-ci.com/docs/tools/php/php-scrutinizer/#list-dependency-paths
```suggestion
    interface TranslatorInterface extends /** @scrutinizer ignore-type */ \Symfony\Component\Translation\TranslatorInterface
```